examples/code_with_line_numbers.md

```diff
@@ -0,0 +1,11 @@
+--- code ---
```

Comment:
Do we need a test for line numbers & highlights?

Reply:
I didn't think we'd need both in one test, but I'll get it done anyway since it's nice and easy to do! Bit of peace of mind and all that.